### PR TITLE
Update internationalization.md

### PR DESCRIPTION
**Pull request by SunlightBro**
fix typo in plural example
https://github.com/flutter/website/issues/8146

### CODE REVIEW

**Review by sfshaza2 — APPROVED** [review]
Thanks for fixing, @SunlightBro!

File: src/development/accessibility-and-localization/internationalization.md [hunk 1]
```diff
@@ -422,12 +422,12 @@ Using a plural method is easy enough, just pass it the item count parameter:
...
-    // Returns 'she'
-    Text(AppLocalizations.of(context)!.pronoun('female')),
-    // Returns 'they'
-    Text(AppLocalizations.of(context)!.pronoun('other')),
+    // Returns 'no wombats'
+    Text(AppLocalizations.of(context)!.nWombats(0)),
+    // Returns '1 wombat'
+    Text(AppLocalizations.of(context)!.nWombats(1)),
+    // Returns '2 wombats'
+    Text(AppLocalizations.of(context)!.nWombats(2)),
```
Comment (parlough) [inline src/development/accessibility-and-localization/internationalization.md:430]: Update these to `5` to match the source snippet:

```suggestion
    // Returns '5 wombats'
    Text(AppLocalizations.of(context)!.nWombats(5)),
```

**Review by parlough — CHANGES_REQUESTED** [review]
Hey @SunlightBro, thanks for noticing and fixing this!

The build is failing since this snippet is updated by pulling from a [Dart file](https://github.com/flutter/website/blob/main/examples/internationalization/gen_l10n_example/lib/main.dart#L68) with the instructions from the code excerpt tag (on line 419). In this case, you need to adjust the code-excerpt instructions to remove the pronoun examples instead of the plural ones. I think something like this would work fine:

```
<?code-excerpt "gen_l10n_example/lib/main.dart (Placeholder)" remove="/John|he|she|they|pronoun/" replace="/\[/[\n    .../g"?>
```

You can make the change or I can if you'd prefer, just let me know :)

You can learn more about at [Code excerpts](https://github.com/dart-lang/site-shared/blob/main/doc/code-excerpts.md) if you'd like.

**Review by parlough — APPROVED** [review]
Thanks so much! Looks good to me :)

**Pull request merged**